modules/virt-using-windows-dvd-disk-image.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-using-windows-dvd-disk-image_{context}"]
+= Using a Windows DVD to create a VM disk image
+
+Microsoft does not provide disk images for download, but you can create a disk image using the Windows DVD. This disk image can then be used to create virtual machines.
+
+.Procedure
+
+. In the {VirtProductName} web console, click *Storage* -> *PersistentVolumeClaims* -> *Create PersistentVolumeClaim With Data upload form*.
+. Select the intended project (e.g. default to select the local windows DVD image).
```

Comment:
"Select the project, for example, the project which should own the Windows VMs"
@awels Can you please help us to decide how the user should decide about the project to upload a disk image?

Reply:
I am not quite sure I understand the question. The UI should have a drop down with projects the user has access to, and they can pick from there. Or if they are using the cli they can add -n _project name_ to the command and it will upload to that project.

Reply:
@awels I just wonder if we can give the user a guidance in selecting the namespace, like  "if the admin wants that all users can read, but not write the image, the image should be uploaded in the openshift-virtualization-os-images project" But maybe it is more wise to give no hints here.

Reply:
It looks like giving hints can get us too far into the weeds, so I'll remove the hints in this procedure instead.